r-pkg/.Rbuildignore

```diff
@@ -1,7 +0,0 @@
-^.*\.DS_Store$
-
-^docs$
-^_pkgdown\.yml$
-cran-comments.md
-tests/testthat/test-integration_tests.R
-inst/testdata/shakespeare_mapping.json
```

Comment:
Can you restore these lines in the new .Rbuildignore? I think they got lost

Reply:
oh weird, I'm surprised that this got overwritten

I'm going to be unhappy if I learn that `pkgdown` overwrote this >.<

Reply:
ugh yeah I'm pretty sure `pkgdown` overwrote this file. Worst